crates/node/src/commands/run.rs

```diff
@@ -45,6 +45,10 @@ pub struct ExtendedRunCmd {
     #[clap(long)]
     pub da_layer: Option<DaLayer>,
 
+    /// Choose a supported DA Layer
+    #[clap(long)]
+    pub da_conf: Option<String>,
```

Comment:
Make them an arg group. 
The whole group is optional, but as soon as one is used, all must be